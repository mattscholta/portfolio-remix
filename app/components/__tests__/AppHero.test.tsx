import { expect, test } from "vitest";
import { render } from "@testing-library/react";

import { AppHero } from "../AppHero";

describe("AppHero Component", () => {
  const component = render(
    <AppHero copy="Test Copy" highlight="The HIGHLIGHT" tag="h2" />
  );

  test("should render our Hero", () => {
    expect(component.baseElement).toMatchSnapshot();
  });
});
