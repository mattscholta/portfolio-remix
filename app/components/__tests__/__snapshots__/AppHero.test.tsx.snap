// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppHero Component > should render our Hero 1`] = `
<body>
  <div>
    <div
      class="hero p-4 text-center"
    >
      <h2
        class="inline-block font-font-serif text-xl font-extrabold md:text-4xl"
      >
        <div
          class="font-font-monospace text-base font-normal md:text-2xl"
        >
          Test Copy
        </div>
        <div
          class="text-highlight px-3 text-4xl md:text-7xl"
        >
          The HIGHLIGHT
        </div>
      </h2>
    </div>
  </div>
</body>
`;
